main.js

```diff
@@ -0,0 +1,312 @@
+import Keys from './components/Keys.js';
+import arrayWithKeys from './components/data.js';
+import fragment from './components/Markdown.js';
+
+let capsLock = true;
+let lang;
+
+if (localStorage.lang === undefined) {
+  lang = 'en';
```

Comment:
'en' is a constant and should be in uppercase at the beginning of the file

Reply:
fixed